scripts/run_e2e.sh

```diff
@@ -4,6 +4,6 @@ set -e
 
 cd e2e-tests/
 
-RUST_LOG=aleph_e2e_client=info,aleph-client=info cargo run -- --node ws://127.0.0.1:9943
+E2E_CONFIG="--node 127.0.0.1:9943" RUST_LOG=aleph_e2e_client=info,aleph-client=info cargo test -- --nocapture
```

Comment:
1) Is it running it in `debug` by purpose? I think it makes sense to leave it that way, as this script is used for running locally by a developer, in which case having not `debug` cause many stack frames to be inlined and emitted from the output in case of error
2) please use `--node ws://127.0.0.1:9943` , `ws://` is required by `subxt` atm
3) thanks to splendid `subxt` migration, we just need ` RUST_LOG=info` as all the `substrate_api_client` raw decoding/encoding logs are no longer there

Reply:
Re 1: I agree it should run in `debug` by default.